inception/inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/LayerStatistics.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.search;
+
+public class LayerStatistics
+{
+    /** The stats to be calculated */
+    public static final String STATS = "n,sum,min,max,mean,median,standarddeviation";
+
+    private long total;
+    private long maximum;
+    private long minimum;
+    private double mean;
+    private double median;
+    private double standardDeviation;
+
+    private double maximumPerSentence;
+    private double minimumPerSentence;
+    private double meanPerSentence;
+    private double medianPerSentence;
+    private double standardDeviationPerSentence;
+
+    private long noOfDocuments;
+
+    public LayerStatistics()
+    {
+    }
+
+    public LayerStatistics(long aTotal, long aMaximum, long aMinimum, double aMean, double aMedian,
+            double aStandardDeviation, double aMaximumPerSentence, double aMinimumPerSentence,
+            double aMeanPerSentence, double aMedianPerSentence,
+            double aStandardDeviationPerSentence, long aNoOfDocuments)
+    {
+        total = aTotal;
+        maximum = aMaximum;
+        minimum = aMinimum;
+        mean = aMean;
+        median = aMedian;
+        standardDeviation = aStandardDeviation;
+
+        maximumPerSentence = aMaximumPerSentence;
+        minimumPerSentence = aMinimumPerSentence;
+        meanPerSentence = aMeanPerSentence;
+        medianPerSentence = aMedianPerSentence;
+        standardDeviationPerSentence = aStandardDeviationPerSentence;
+
+        noOfDocuments = aNoOfDocuments;
+    }
+
+    public boolean isLayerTrivial()
```

Comment:
Can you add a comment what this does / why its needed?
also, its just `return maximum == 0L;`

Reply:
Well, if the maximum is 0 then there is no document which has an annotation from this feature so all statistics for this feature are 0. 
It was useful for the previous approach where I cycled through all of the features. I can remove it together with the nonTrivialResults from StatisticsResult.

Reply:
Removed it.